src/libraries/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -295,7 +299,7 @@
              Link="Common\Interop\Windows\Interop.HRESULT.cs" />
   </ItemGroup>
   <!-- Unix specific -->
-  <ItemGroup Condition="'$(TargetsUnix)' == 'true' or '$(TargetsBrowser)' == 'true'">
+  <ItemGroup Condition="'$(TargetsUnix)' == 'true' and '$(TargetsBrowser)' != 'true'">
```

Comment:
We don't need this and as `TargetsBrowser` is no longer part of `Unix` right?

Reply:
Correct

Reply:
Fixed